<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitPaddockDialogBase</class>
 <widget class="QDialog" name="SplitPaddockDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Split Paddock</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="fenceLengthLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Estimated fence length (m)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fenceLengthText">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelSplitButton">
        <property name="text">
         <string>Cancel Split</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="confirmSplitButton">
        <property name="text">
         <string>Confirm Split</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <connections/>
</ui>
